server/notification-providers/wecom.js

```diff
@@ -30,26 +31,50 @@ class WeCom extends NotificationProvider {
     /**
      * Generate the message to send
      * @param {object} heartbeatJSON Heartbeat details (For Up/Down only)
+     * @param {object} monitorJSON Monitor details
      * @param {string} msg General message
      * @returns {object} Message
      */
-    composeMessage(heartbeatJSON, msg) {
-        let title;
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === UP) {
-            title = "UptimeKuma Monitor Up";
-        }
-        if (msg != null && heartbeatJSON != null && heartbeatJSON["status"] === DOWN) {
-            title = "UptimeKuma Monitor Down";
+    composeMessage(heartbeatJSON, monitorJSON, msg) {
+        let params = {};
+        if (heartbeatJSON != null) {
+            params = {
+                msgtype: "markdown",
+                markdown: {
+                    content: `## [${this.statusToString(
+                        heartbeatJSON["status"]
+                    )}] ${monitorJSON["name"]} \n> ${
+                        heartbeatJSON["msg"]
+                    } \n> Time (${heartbeatJSON["timezone"]}): ${
+                        heartbeatJSON["localDateTime"]
+                    }`,
+                },
+            };
+        } else {
+            params = {
+                msgtype: "text",
+                text: {
+                    content: msg,
+                },
+            };
         }
-        if (msg != null) {
-            title = "UptimeKuma Message";
+        return params;
+    }
+
+    /**
+     * Convert status constant to string
+     * @param {const} status The status constant
+     * @returns {string} Status
+     */
+    statusToString(status) {
+        switch (status) {
+            case DOWN:
+                return "DOWN";
```

Comment:
Is there a reason you removed the emojis?

Reply:
no particular reasons, just thought emoji in the title would be distracting.

Reply:
Adding the emojis has the advantage that people can easier differentiate between good (`UP`) and bad `DOWN` messages